openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -54,7 +54,6 @@
 from openedx.core.lib.api.parsers import MergePatchParser
 from common.djangoapps.student.models import (  # lint-amnesty, pylint: disable=unused-import
```

Comment:
This warning suppression should be removed now.
```suggestion
from common.djangoapps.student.models import (
```

Reply:
Removing it raised error